client/src/services/connectedCalendars/calendarService.js

```diff
@@ -3,5 +3,11 @@ import requestService from '@/services/requestService';
 export default {
     async connect(calendar) {
         return await requestService.get(`/social-accounts/${calendar}/oauth`);
+    },
+
+    async disconnect(provider) {
+        return await requestService.delete(
+            `/social-accounts/calendar/${provider}`
```

Comment:
Can you add `/social-accounts/` to constant and use it everywhere in this file?

Reply:
done